src/Collections-Unordered/Bag.class.st

```diff
@@ -104,7 +137,11 @@ Bag >> cumulativeCounts [
 
 { #category : #enumerating }
 Bag >> do: aBlock [ 
-	"Evaluate aBlock with each of the receiver's elements as the argument."
+	"Evaluate aBlock with each of the receiver's elements as the argument.
+	
+	This will print all of the occurrences of the value. Use doWithOccurrences: 
```

Comment:
Hello I do not get the this will print...
Because do: does not print anything.

Reply:
Yes. This was poor wording on my part. I replaced with "enumerate"